### PR TITLE
DT-961 Instructions for testing and new auth urls

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +56,72 @@ Running all services except this application (hence allowing you to run this in
...
 ```
 
+Depending on the speed of your machine when running all services you may need to scale `probation-offender-search-indexer=0` until localstack starts. This is a workaround for an issue whereby Spring Boot gives up trying to connect to SQS when the services first starts up.
+
+### Running tests
+
+#### Test containers
+
+`./gradlew test` will run all tests and will by default use test containers to start any required docker containers, e.g localstack
+Note that TestContainers will start Elastic Search in its ow n container rather than using the one built into localstack.
```
Comment (brightonsbox) [inline README.md:66]: meh... small typo (ow n -> own)

File: README.md [hunk 2]
```diff
@@ -53,6 +56,72 @@ Running all services except this application (hence allowing you to run this in
...
+      "status": "UP",
+      "details": {
+        "MessagesOnQueue": 41834,
+        "MessagesInFlight": 4,
+        "dlqStatus": "UP",
+        "MessagesOnDLQ": 0
+      }
+    }
+```
+would be a valid state
```
Comment (brightonsbox) [inline README.md:112]: maybe `would be a valid state since "MessagesOnDLQ" is zero`?

File: README.md [hunk 3]
```diff
@@ -53,6 +56,72 @@ Running all services except this application (hence allowing you to run this in
...
+
+The build can either left to run cancelled using
```
Comment (brightonsbox) [inline README.md:114]: Meh... The build can either **be** left to run **or** cancelled using

File: README.md (hunk 1 above)
Comment (andymarke) [reply README.md]: yep, corrected

File: README.md (hunk 2 above)
Comment (andymarke) [reply README.md]: yep, corrected

File: README.md (hunk 3 above)
Comment (andymarke) [reply README.md]: yep, corrected

**Pull request merged**